Products/CMFPlone/tests/robot/test_controlpanel_site.robot

```diff
@@ -94,8 +94,13 @@ then I can see a sitemap
 
     Import library    OperatingSystem
 
-    Download    ${PLONE_URL}/sitemap.xml.gz    saveAs=/tmp/sitemap.xml.gz
-    File Should Exist    /tmp/sitemap.xml.gz
+    # this is for robotframework browser > 17
+    # Download    ${PLONE_URL}/sitemap.xml.gz    saveAs=/tmp/sitemap.xml.gz
```

Comment:
We should go with the version >=18 and add it as test requirement here. Due to the fact that the tests are flaky event after backporting the version I'd rather merge the updated version in coredev (see https://github.com/plone/buildout.coredev/pull/954) and continue getting this here done ... @mauritsvanrees what do you think?

Reply:
I updated the test to the rfb 18 version